project/MicroService.scala

```diff
@@ -82,7 +82,8 @@ trait MicroService {
       Wart.Nothing,
       Wart.Overloading,
       Wart.ToString,
-      Wart.Var)
+      Wart.Var,
+      Wart.NonUnitStatements)
```

Comment:
why do we need to ignore `NonUnitStatements`?

Reply:
its Wart is complaining about this because I used `.map(_ ⇒ ())` in UserCapStore

Reply:
I enabled `NonUnitStatements` again and it seems to be fine